machines/bleShare/scan/scanMachine.ts

```diff
@@ -113,7 +117,9 @@ const model = createModel(
       UPDATE_VC_NAME: (vcName: string) => ({vcName}),
       STORE_RESPONSE: (response: any) => ({response}),
       APP_ACTIVE: () => ({}),
-      FACE_VALID: () => ({}),
+      FACE_VALID: (isFaceIDVerificationFlow: boolean) => ({
```

Comment:
Same here, as we are using FACE_VALID in only one place with value as true 
https://github.com/mosip/inji/pull/1270/files#diff-8644aa2a28639628e4d0ed0920d026bd4470b5b24616264cc144401f39332192R68

Reply:
Removed the param and passing the data directly now.